composer.json

```diff
@@ -2,7 +2,8 @@
     "name": "dev-community/code-formatter",
     "description": "A simple web app to format code in XenForo forum posts",
     "require": {
-        "php": ">=7.3"
+        "php": ">=7.3",
+        "phan/phan": "^3.0"
```

Comment:
@asccc 
Why have we added phan to the regular dependencies rather than `require-dev` like with the Symfony var-dumper?
I mean, we don't want phan in our production environment.
Can I safely change that or is there any good reason to keep it like that?

Reply:
Yes, this was my fault

Reply:
Ok, I will change that. Thanks for the quick response. 👍